.github/workflows/build-oci.yaml

```diff
@@ -67,7 +66,28 @@ jobs:
         with:
           name: mapt-${{ env.ARCH_TYPE }}
           path: mapt*
+  combine-artifacts:
+    name: combine-artifacts
+    needs: build-and-upload
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
+      - name: Download mapt oci flatten images
+        uses: actions/download-artifact@cc203385981b70ca67e1cc392babf9cc229d5806 # v4.1.9
+        with:
+          pattern: mapt-*
+      - name: copy both artifacts into single directory
+        run: |
+          mkdir mapt-arm64-and-amd64
+          mv mapt-amd64/* mapt-arm64-and-amd64/
+          mv mapt-arm64/* mapt-arm64-and-amd64/
+      - name: Upload combined mapt artifacts for PR
+        uses: actions/upload-artifact@4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1 # v4.6.1
+        with:
+          name: mapt-arm64-and-amd64
+          path: mapt-arm64-and-amd64/
```

Comment:
you can use pattern instead of create a folder and added them there...then even the download part would be easier as they would be uncompressed on the current path there and you will have both tars... 

actually you are doing `mapt-amd64/*` b/c inside there is tar and image but image has the same value on both....

so probably you can grab the image value within this job....and remove it from builders....

then you would have something like:

mapt-amd64.tar
mapt-arm64.tar
mapt-image

and you can upload with patter: `mapt-*`